plugins/nv-filters/data/locale/en-US.ini

```diff
@@ -0,0 +1,17 @@
+Nvafx="NVIDIA Audio Effects"
+Nvafx.SuppressLevel="Suppression Level"
+Nvafx.Intensity="Suppression Intensity"
+Nvafx.Method="Method"
+Nvafx.Method.Denoiser="NVIDIA Noise Removal"
+Nvafx.Method.Dereverb="NVIDIA Room Echo Removal"
+Nvafx.Method.DenoiserPlusDereverb="NVIDIA Noise Removal + Room Echo Removal"
+Nvafx.OutdatedSDK="WARNING: Please upgrade both NVIDIA Video & Audio SDK. Your current version of Audio SDK is outdated."
+Nvvfx.Method.Greenscreen="NVIDIA Background Removal"
+Nvvfx.Method.Greenscreen.Mode="Mode"
+Nvvfx.Method.Greenscreen.Quality="Quality (higher GPU usage, better quality)"
+Nvvfx.Method.Greenscreen.Performance="Performance (lower GPU usage, good quality)"
+Nvvfx.Method.Greenscreen.Threshold="Threshold"
+Nvvfx.OutdatedSDK="WARNING: Please upgrade both NVIDIA Video & Audio SDK. Your current version of Video SDK is outdated."
+Nvvfx.Method.Greenscreen.Processing="Mask refresh frequency in frames"
+Nvvfx.Method.Greenscreen.Processing.Hint="This alleviates GPU load by generating a mask every N frames only (2 on default)."
+
```

Comment:
No need to end the file with two blank lines, just one.

Reply:
fixed